src/libraries/System.Security.Cryptography.Algorithms/src/System/Security/Cryptography/RandomNumberGenerator.cs

```diff
@@ -146,6 +146,29 @@ public static int GetInt32(int toExclusive)
             return GetInt32(0, toExclusive);
         }
 
+        /// <summary>
+        /// Creates an array of bytes with a cryptographically strong random sequence of values.
+        /// </summary>
+        /// <param name="byteCount">The number of bytes of random values to create.</param>
+        /// <returns>
+        /// An array populated with cryptographically strong random values.
+        /// </returns>
+        /// <exception cref="ArgumentOutOfRangeException">
+        /// The <paramref name="byteCount" /> parameter is less than zero.
+        /// </exception>
+        public static byte[] GetBytes(int byteCount)
+        {
+            if (byteCount == 0)
```

Comment:
Honestly I'd remove this if check. There's no point to optimizing the zero-length case because nobody will realistically ever pass 0 as a parameter.